WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/BloggingPromptsAttribution.swift

```diff
@@ -45,8 +63,12 @@ enum BloggingPromptsAttribution: String {
             .font: WPStyleGuide.fontForTextStyle(.caption1, fontWeight: .medium),
             .foregroundColor: UIColor.label,
         ]
-        static let iconSize = CGSize(width: 18, height: 18)
-        static let dayOneIcon = UIImage(named: "logo-dayone")?.resized(to: Constants.iconSize)
+        static let dayOneIconSize = CGSize(width: 18, height: 18)
+        static let dayOneIcon = UIImage(named: "logo-dayone")?.resized(to: Constants.dayOneIconSize)
+        static let dayOneURL = URL(string: "https://dayoneapp.com/?utm_source=jetpack&utm_medium=prompts")
+
+        static let linkIconSize = CGSize(width: 12, height: 12)
+        static let linkIcon = UIImage.gridicon(.external, size: Constants.linkIconSize)
```

Comment:
(nit) I'd suggest using an SF symbol like "link" or one of the image from the WordPress Design Library. We are slowly removing `Gridicon` usages in the app.